sources/main.cpp

```diff
@@ -123,13 +122,8 @@ int main(int argc, char** argv) {
 
       boost::shared_ptr<DYN::IoDicos> dicos = DYN::IoDicos::getInstance();
       dicos->addPath(getMandatoryEnvVar("DYNAWO_RESOURCES_DIR"));
-      std::string locale = getMandatoryEnvVar("DYNAWO_ALGORITHMS_LOCALE");
-      dicos->addDico("CONSTRAINT", "DYNConstraint", locale);
-      dicos->addDico("ERROR", "DYNError", locale);
-      dicos->addDico("LOG", "DYNLog", locale);
-      dicos->addDico("TIMELINE", "DYNTimeline", locale);
-      dicos->addDico("AlgorithmsERROR", "DYNAlgorithmsError", locale);
-      dicos->addDico("AlgorithmsLOG", "DYNAlgorithmsLog", locale);
+      dicos->addDicos(getMandatoryEnvVar("DYNAWO_DICTIONARIES"),
```

Comment:
I don't really like the fact that we use DYNAWO_DICTIONARIES (that is the same in dynawo) rather than DYNAWOALGORITHM_DICTIONARIES for example. For me this is confusing if the env var is used only by this main or later on by dynawo also. Also here I see 6 entries added manually but I only see 2 mappings in the mapping file above, is it ok ?

Reply:
I have consider DYNAWO_DICTIONARIES the same way DYNAWO_RESOURCES_DIR is used and DYNAWO_DICTIONARIES was already initialized in envDynawoAlgorithms.sh !

All dictionaries_mapping.dic from all path in DYNAWO_RESOURCES_DIR are combined : so these 2 entries complete the 4 ones already defined in Dynawo.

Reply:
Ok things are clear now and you can forget my comment !